Components/DayCalendar.js

```diff
@@ -9,50 +12,251 @@ var {
   ScrollView,
   View,
   Text,
+  PanResponder,
+  Image,
 } = React;
 
+type Event = {
+  title: string;
+  startAt: Date;
+  length: number;
+};
```

Comment:
I guess the packager asks flow to strip all the annotations before feeding the code to babel.